### DISCUSSION TITLE
Pore Size Distribution

### DISCUSSION THREAD

**Discussion by HFenice** [PMEAL/OpenPNM#1979] [Q&A] [answered]
Hello everyone,

is it possible to create a Pore Network Model by starting from a Pore Size Distribution imported in openpnm?

Thank you in advance.

**Comment by jgostick** [comment, marked as answer]
Hi @HFenice 
There is an example notebook that outlines how you can adjust the pore/throat size distributions to match the behavior of a material using a cubic network.  

https://github.com/PMEAL/OpenPNM/blob/dev/examples/notebooks/geometry/adjusting_pore_size_distributions.ipynb

**Reply by HFenice** [reply]
Many thanks!

**Reply by HFenice** [reply]
Once this pore network has been created, is it possible to import it in Avizo?

**Reply by jgostick** [reply]
Actually yes!  Use the PerGeos class.  This was developed informally with someone from Avizo a few years ago. As far as I know it still works.
